js/classes/entities/midground/npc/village/papa_chad.js

```diff
@@ -66,10 +65,6 @@ class PapaChad {
         return "./sprites/parents.png";
     };
 
-    /** The width, in pixels, of the sprite ON THE SPRITESHEET. */
-    static get WIDTH() {
-        return 29;
-    };
     /** The velocity of the first jump */
```

Comment:
All of these getters - FIRST_JUMP_VELOCITY thru DASH_BARRIER - ought be static. Their return does not depend upon the state of this {Papa}Chad instance - therefore, they should be static and called by saying Chad.FIRST_JUMP_VELOCITY, not CHAD.FIRST_JUMP_VELOCITY